flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+	"only-arches": ["x86_64"]
```

Comment:
why?

Reply:
The application doesn't build on aarch64 (https://buildbot.flathub.org/#/builders/30/builds/4208, although not a flatpak specific issue). I was unaware of this until I submitted the PR, since I use GitHub Actions and they don't have any aarch64 runners. I can look into fixing this in a future version, but in its current state, only x86_64 actually works.